docs/source/guides/tools/MDLModding.rst

```diff
@@ -0,0 +1,307 @@
+MDL Modding
+===========
+
+MDL Overview
+------------
+
+Model (MDL) is the extension for Source's proprietary model format. It defines the structure of the model along with animation, bounding box, hit box, materials, mesh and LOD information. Unlike other Source games in Titanfall 2 files like the .phy, .vtx, .vvd etc. are not separate files and are instead included in the .mdl file.
+
+`VALVe developer docs "Model" <https://developer.valvesoftware.com/wiki/.mdl>`__
+
+
+Editing an existing Model
+-------------------------
+
+While creating an entire Model from scratch is possible it would be an extremely long and tedious task. Instead its recommended to use an existing Model as your base.
+
+
+Prequisites
+-----------
+•	`Blender <https://www.blender.org/download/>`__ (min. 2.92)
+•	`Blender Source Tools <https://developer.valvesoftware.com/wiki/Blender_Source_Tools>`__  (refer to Download page for install instructions)
+•	`Crowbar <https://steamcommunity.com/groups/CrowbarTool>`__ (min 0.74)
+
+        - One of the following two games on Steam:
+
+        - `Portal 2 <https://store.steampowered.com/app/620/Portal_2/>`__ (and its Authoring Tools `Portal 2 SDK <https://developer.valvesoftware.com/wiki/Authoring_Tools/SDK_(Portal_2)>`__)  
+
+        - `SourceFilmMaker <https://store.steampowered.com/app/1840/Source_Filmmaker/>`__ (FREE)
+
+        - `Alien Swarm <https://store.steampowered.com/app/630/Alien_Swarm/>`__ (FREE) (and its Authoring Tools `Alien Swarm SDK <https://developer.valvesoftware.com/wiki/Authoring_Tools/SDK_(Alien_Swarm)>`__)
```

Comment:
two games?

Reply:
Technically SFM isn't a game (but yeah this should probably be fixed)

Reply:
fixed